internal/impl/azure/integration_test.go

```diff
@@ -123,7 +123,7 @@ output:
     blob_type: BLOCK
     container: $VAR1-$ID
     max_in_flight: 1
-    path: $VAR2/${!count("$ID")}.txt
+    path: $VAR2/${!counter()}.txt
```

Comment:
It probably doesn't make a difference here, but I guess in such cases where one wants a new counter for every new value of `$ID`, they should use the old `count()`.

Reply:
It actually doesn't change anything (I think) because the config is instantiated once per test anyway, so each counter will still be unique to the ID. I can't really remember when these tests were written though, but they pass so I'm over it 😅